### ISSUE TITLE
please add support for (plugin).value "U"

### ISSUE THREAD

**Issue by solact** [smdn/PseudoMuninNode#2]
I would like to return "U" as a (plugin).value when the plugin failed, but there is no way to return "U".
http://guide.munin-monitoring.org/en/latest/reference/plugin.html

> Attribute: {fieldname}.value
> Value: integer, decimal numbers, or “U” (may be signed). For DERIVE and COUNTER values this must be an integer. See rrdcreate for restrictions.

Is there any good way to do that? My idea is like this.

change this line
https://github.com/smdn/PseudoMuninNode/blob/master/src/PseudoMuninNode.cpp#L180
`writeResponse(field->name + ".value " + isnan(field->value) ? "U" : field->value, client);`
and plugin returns std::numeric_limits\<double\>::quiet_NaN() when it failed.

Any better ideas are welcome.

**Comment by smdn** [comment]
@solact Thank you for your feedback!

In my opinion, it is not appropriate to use NaN for `value` to determine whether to return "U" or not.
This is because NaN will represent two different meanings: the simple meaning of NaN, and the meaning of "U".
And this will make it impossible to determine the difference between them.

So, my idea is adding a new field like `isValueUnknown` instead.
The code will like below.

```c
if (get_sensor_value(current_sensor_value)) {
  // If a value can be retrieved from the sensor, set that value.
  field[0].value = current_sensor_value;
  field[0].isValueUnknown = false;
}
else {
  // If a value cannot be retrieved from the sensor, set to "U".
  field[0].isValueUnknown = true;
}
```

```c
// returns "U" if isValueUnknown is true, otherwise just returns field value
writeResponse(field->name + ".value " + field->isValueUnknown ? "U" : field->value, client);
```

If this achieves your goal, I will modify the code in this way.
How do you think?

**Comment by solact** [comment]
@smdn Thank you for very quick reply!

I read your idea, and it seems to be more accurate way. I'm too worried about using more memory :-)
Your sample code does what I want to do, could you modify your code like that sample?

Finally, thank you for this project. Nice work!

**Comment by smdn** [comment]
Modified code and released version [1.0.2](https://github.com/smdn/PseudoMuninNode/releases/tag/v1.0.2).

Also example sketch to report "U" value is added.
The sketch is available here: https://github.com/smdn/PseudoMuninNode/blob/master/examples/04_ReportUnknownValue/04_ReportUnknownValue.ino

Please update and use it. Thank you!
